pkg/proto/configuration/global/global.proto

```diff
@@ -56,18 +57,95 @@ message PrometheusPushgatewayConfiguration {
   google.protobuf.Duration push_interval = 5;
 }
 
+message OpenCensusConfiguration {
+  // Export data to an OpenCensus agent. The recommended configuration
+  // is to use opentelemetry-collector and enable its opencensus-agent
+  // receiver.
+
+  // Connection address for the agent. Default is localhost:55678.
+  string address = 1;
+
+  // TLS configuration. TLS is not enabled when left unset.
+  buildbarn.configuration.tls.ClientConfiguration tls = 2;
+
+  // OpenCensus service name.
+  string service_name = 3;
+}
+
+message OpenTelemetryConfiguration {
+  // Connection address for the collector. Default is localhost:55680.
+  string address = 1;
+
+  // TLS configuration. TLS is not enabled when left unset.
+  buildbarn.configuration.tls.ClientConfiguration tls = 2;
+
+  // Resource detectors. Labels will be merged, with earlier things in
+  // this entries being preferred over later ones.
+  enum ResourceDetector {
+    ENV = 0;  // Use OTEL_RESOURCE_ATTRIBUTES environment variable
```

Comment:
Do we need to use names this short? I'm fine with abbreviating "Amazon Web Services" to just AWS, but at least this configuration option could use a name that's more descriptive. USE_ENVIRONMENT_VARIABLES?

Reply:
It's not necessary, I was just matching the names of the detectors in the otel library. Changed.